dflag/configmap/updater_test.go

```diff
@@ -124,7 +124,8 @@ type (
 
 // eventually tries a given Assert function 5 times over the period of time.
 func eventually(t *testing.T, duration time.Duration,
-	af assertFunc, expected interface{}, actual getter, msgFmt string, msgArgs ...interface{}) {
+	af assertFunc, expected interface{}, actual getter, msgFmt string, msgArgs ...interface{},
+) {
```

Comment:
how come you got formatting changes here? undo? (I don't think gofumpt complains about this file?)

Reply:
reverted. I run the `golangci-lint run --fix ` locally and this file got changed.

Reply:
ic, you have to make sure to use the version in the build image, which may not be same as on your mac